plugin.xml

```diff
@@ -5,12 +5,13 @@
   See the accompanying LICENSE file for applicable licenses.
 -->
 <plugin id="net.infotexture.dita-bootstrap.lunr" version="1.0.0">
-  <require plugin="net.infotexture.dita-bootstrap"/>
+  <require plugin="net.infotexture.dita-bootstrap" />
```

Comment:
This is consistent with the formatting of the surrounding lines, but in the main `dita-bootstrap` plug-in, we [enforce](https://github.com/infotexture/dita-bootstrap/blob/develop/.prettierrc.json#L15) the spaceless syntax with `"xmlSelfClosingSpace": false`, so we may want to align formatting here.

— But I can do that in a separate PR to keep things focused here, so fine for now.

Reply:
Sounds like a separate PR, create a `.prettierrc.json` file and run it over everything.

Reply:
Yep, will do.